R/profile_plot.R

```diff
@@ -18,6 +18,7 @@
 #' @seealso [profile_wrapper] and [rerun_profile_vals] call `profile_plot`.
 
 profile_plot <- function(mydir, rep, para, profilesummary) {
+  
```

Comment:
Remove the space.

Reply:
I intentionally added this blank line. I suspect you recommending this based on a formatting guide. For me, I find the code more difficult to read without a blank line between the function inputs and the first line within the function. How attached are you to the removal of the blank line?

Reply:
Not attached at all.